react-native-libraries.json

```diff
@@ -5255,15 +5255,18 @@
     "expo": true
   },
   {
+    "npmPkg": "@testing-library/react-native",
+    "nameOverride": "React Native Testing Library",
     "githubUrl": "https://github.com/callstack/react-native-testing-library",
     "examples": [
       "https://github.com/callstack/react-native-testing-library#example",
       "https://github.com/callstack/react-native-testing-library/tree/master/examples/reactnavigation",
       "https://github.com/callstack/react-native-testing-library/tree/master/examples/redux"
     ],
+    "images": ["https://callstack.github.io/react-native-testing-library/img/owl.png"],
```

Comment:
Thank you for the update @mdjastrzebski! 🙏 

```suggestion
```

Let's remove asset since it does not represent functionality:
* https://github.com/react-native-community/directory#images

Besides that, LGMT! 👍